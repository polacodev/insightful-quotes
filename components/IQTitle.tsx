import React from "react";
import { View, Text } from "react-native";

const IQTitle = () => {
  return (
    <View>
      <Text style={{ fontFamily: 'Pacifico-Regular', fontSize: 18 }} className="text-[#333] dark:text-[#bae4fd]">Insightful Quotes</Text>
    </View>
  );
};

export default IQTitle;
