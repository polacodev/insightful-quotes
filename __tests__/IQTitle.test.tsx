import { render } from "@testing-library/react-native";

import IQTitle from "@/components/IQTitle";

describe("<IQTitle />", () => {
  test("IQTitle renders correctly the title and Snapshot", () => {
    const { getByText, toJSON } = render(<IQTitle />);
    const title = getByText("Insightful Quotes")

    expect(title).toBeTruthy();
    expect(toJSON()).toMatchSnapshot();
  });
});
