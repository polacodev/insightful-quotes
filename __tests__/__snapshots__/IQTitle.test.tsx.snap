// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IQTitle /> IQTitle renders correctly the title and Snapshot 1`] = `
<View>
  <Text
    className="text-[#333] dark:text-[#bae4fd]"
    style={
      {
        "fontFamily": "Pacifico-Regular",
        "fontSize": 18,
      }
    }
  >
    Insightful Quotes
  </Text>
</View>
`;
